lib/webtask.js

```diff
@@ -62,6 +65,10 @@ exports.validate_token_issue_request = function(req, claims) {
         return { code: 400, message: 'specific jtn may only be requested when issuing tokens to be used in specific containers' };
     }
 
+    if(req.auth && !req.ten) {
```

Comment:
Do we still need `auth` claim? What's inside? 

I think the design we discussed on Slack had us add *no* new claims, but use a pattern of adding two pieces of data to `ectx`: the JWT secret and expected JWT aud. Given @jaredhanson's latest comments of JWT secret is present (which triggers the authentication behavior), the JWT aud also MUST be present. 

The JWT secret must of course be encrypted, which can be done by simply putting it in ectx. If we choose to do that, I think the the names of these two properties should start with `webtask`, e.g. `webtask_jwt_secret` and `webtask_jwt_aud` - we make all properties starting with `webtask_` reserved names. 

Another way to pass these properties would be to actually introduce a new claim (and `auth` may work just as well), but then we need to augment the logic that encrypts/decrypts that claim the same way we encrypt/decrypt ectx.